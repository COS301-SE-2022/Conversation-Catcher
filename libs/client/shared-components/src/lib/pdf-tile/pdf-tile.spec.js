import React from 'react';
import { render } from '@testing-library/react-native';
import PdfTile from './pdf-tile';
describe('PdfTile', () => {
  it('should render successfully', () => {
    const { container } = render(<PdfTile />);
    expect(container).toBeTruthy();
  });
});
