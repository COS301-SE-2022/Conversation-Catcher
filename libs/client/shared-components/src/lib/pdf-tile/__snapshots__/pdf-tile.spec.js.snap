// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PdfTile snapshot 1`] = `
<View
  style={
    Object {
      "flexGrow": 0,
      "flexShrink": 1,
    }
  }
>
  <button
    id="btn"
    onPress={[Function]}
    style={
      Object {
        "borderColor": "#d0d5ddff",
        "borderRadius": 5,
        "borderStyle": "solid",
        "borderWidth": 1,
        "elevation": 2,
        "flexDirection": "row",
        "flexGrow": 1,
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 1,
          "width": 0,
        },
        "shadowOpacity": 0.2173913043478261,
        "shadowRadius": 2.621621621621622,
      }
    }
    type="button"
    x="0px 356fr 0px"
    y="0px minmax(0px, max-content) 0px"
  >
    <View
      style={
        Object {
          "flexBasis": 90,
          "flexGrow": 0,
          "flexShrink": 1,
        }
      }
    >
      <img
        alt="PDF"
        source={
          Object {
            "testUri": "../../../libs/client/ui/src/lib/assets/23a067db3896c1ac956e0e22fe9c7588.png",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 5,
              "resizeMode": "contain",
            },
            Object {
              "height": 127,
              "marginBottom": 0,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "minWidth": 90,
              "width": 90,
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "flexBasis": 13,
          "flexGrow": 0,
          "flexShrink": 1,
        }
      }
    />
    <View
      style={
        Object {
          "flexBasis": 253,
          "flexGrow": 0,
          "flexShrink": 1,
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
            "flexGrow": 1,
          }
        }
        x="0px 253fr 0px"
        y="0px minmax(0px, max-content) 0px"
      >
        <View
          style={
            Object {
              "flexBasis": 220,
              "flexGrow": 0,
              "flexShrink": 1,
            }
          }
        >
          <View
            style={
              Object {
                "flexGrow": 1,
              }
            }
            x="1px 219fr 0px"
            y="0px minmax(0px, max-content) 0px"
          >
            <View
              style={
                Object {
                  "flexGrow": 0,
                  "flexShrink": 1,
                }
              }
            >
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "flexGrow": 1,
                    "justifyContent": "flex-start",
                  }
                }
                x="0px 218fr 1px"
                y="29px minmax(0px, max-content) 0px"
              >
                <Text
                  ellipsizeMode="clip"
                  style={
                    Object {
                      "color": "#344053ff",
                      "fontFamily": "System",
                      "fontSize": 16,
                      "fontStyle": "normal",
                      "fontWeight": "400",
                      "letterSpacing": 0,
                      "lineHeight": 20,
                      "paddingHorizontal": 0,
                      "paddingVertical": 0,
                      "textAlign": "left",
                    }
                  }
                >
                  Bug introduction: a modification of code
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "flexGrow": 0,
                  "flexShrink": 1,
                }
              }
            >
              <View
                style={
                  Object {
                    "alignItems": "flex-end",
                    "flexDirection": "row",
                    "flexGrow": 1,
                    "justifyContent": "flex-start",
                  }
                }
                x="1px 217fr 1px"
                y="9px minmax(0px, max-content) 29px"
              >
                <Text
                  ellipsizeMode="clip"
                  style={
                    Object {
                      "color": "#667084ff",
                      "fontFamily": "System",
                      "fontSize": 14,
                      "fontStyle": "normal",
                      "fontWeight": "400",
                      "letterSpacing": 0,
                      "lineHeight": 20,
                      "paddingHorizontal": 0,
                      "paddingVertical": 0,
                      "textAlign": "left",
                    }
                  }
                >
                  1 May 2022, 9:37
                </Text>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flexBasis": 33,
              "flexGrow": 0,
              "flexShrink": 1,
            }
          }
        >
          <button
            onPress={[Function]}
            style={
              Object {
                "flexGrow": 1,
                "width": "100%",
              }
            }
            x="0px 33fr 0px"
            y="0px minmax(0px, max-content) 89px"
          >
            <img
              alt="PDF"
              source={
                Object {
                  "testUri": "../../../libs/client/ui/src/lib/assets/6784b72f243ed3938b4effbf0ffc0c7a.png",
                }
              }
              style={
                Array [
                  Object {
                    "resizeMode": "contain",
                  },
                  Object {
                    "height": 18,
                    "marginBottom": 10,
                    "marginLeft": 4,
                    "marginRight": 11,
                    "marginTop": 10,
                    "minWidth": 18,
                    "width": 18,
                  },
                ]
              }
            />
          </button>
        </View>
      </View>
    </View>
  </button>
</View>
`;
